conandata.yml

```diff
@@ -607,3 +607,10 @@ python_translation_source_folders:
 qml_translation_source_folders:
   - resources/qml
   - plugins
+
+extra_dependencies:
+  conan:
+    version: "2.7.1"
```

Comment:
I saw you put a comment in the Uranium repo for Qt to change the version in the requirements -- maybe it's a good idea to do something similar here, even if only in one place.

Reply:
True, the conan version is in every recipe, but I can add it at least to the Cura one